app/views/admin/attachments/_attachments.html.erb

```diff
@@ -1,45 +1,37 @@
 <% initialise_script "GOVUK.ReorderableAttachmentsList", el: "ol.existing-attachments" if attachable.can_order_attachments? && attachable.attachments.many? %>
 
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Attachments",
+  font_size: "l",
+  margin_bottom: 3,
+} %>
+
 <% if attachable.can_order_attachments? && attachable.attachments.many? %>
-  <p class="alert alert-info">
-    To change the order of your attachments, drag them up or down and then click ‘Save attachment order’ at
-    the bottom of the page.
-  </p>
+  <%= link_to 'Reorder attachments', reorder_admin_edition_attachments_path(@edition), class: "govuk-link govuk-link--no-visited-state" %>
```

Comment:
This should use a href with the design system classes rather than the bootstrap template.

Reply:
link_to is actually a rails helper https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to rather than a Bootstrap template so i think we're all good here 

```
link_to 'Reorder attachments', reorder_admin_edition_attachments_path(@edition), class: "govuk-link govuk-link--no-visited-state"
```

becomes

```
 <a class="govuk-link govuk-link--no-visited-state" href="/government/admin/editions/1288858/attachments/reorder">Reorder attachments</a>
```

Reply:
Oh, thanks David